src/test/resources/quick/quick.html_metadata.json

```diff
@@ -0,0 +1,5 @@
+{
+  "{http://www.alfresco.org/model/content/1.0}author": "Nevin Nollop",
+  "{http://www.alfresco.org/model/content/1.0}description": "Gym class featuring a brown fox and lazy dog",
+  "{http://www.alfresco.org/model/content/1.0}title": "The quick brown fox jumps over the lazy dog"
+}
```

Comment:
New line at end of file

Reply:
Deliberate. This is exactly what the T-Engine would send.